packages/termux-gui-package/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/tareksander/termux-gui-package"
+TERMUX_PKG_DESCRIPTION="A Termux package containing utilities for Termux:GUI"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@tareksander"
```

Comment:
Most packages use the form `Tarek Sander @tareksander`, and a few `@tareksander` or `Tarek Sander <your-email@email.com>`.

Current form is acceptable.